src/gluonnlp/layers.py

```diff
@@ -356,7 +356,8 @@ def __init__(self, mode='erf'):
 
     def hybrid_forward(self, F, x):
         if self._mode == 'erf':
-            return F.npx.leaky_relu(x, act_type='gelu')
+            # TODO Investigate the precision of F.npx.leaky_relu(x, act_type='gelu')
+            return x * 0.5 * (1.0 + F.npx.erf(x / math.sqrt(2.0)))
```

Comment:
Here is a precision issue related to `F.npx.leaky_relu` as https://github.com/apache/incubator-mxnet/issues/18826. We could revert this back once it solved.